lib/elements/steel_reinforcement/rebar_hook.rb

```diff
@@ -0,0 +1,16 @@
+require 'db/base'
+require 'elements/steel_reinforcement/base'
+
+module Elements
+  module SteelReinforcement
+    class RebarHook < Base
+      attr_accessor :angle
```

Comment:
Let's use `attr_reader` unless completely necessary
```suggestion
      attr_reader :angle
```

Reply:
done!